homeassistant/components/saunum/climate.py

```diff
@@ -0,0 +1,105 @@
+"""Climate platform for Saunum Leil Sauna Control Unit."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pysaunum import MAX_TEMPERATURE, MIN_TEMPERATURE
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import LeilSaunaConfigEntry, LeilSaunaCoordinator
+from .entity import LeilSaunaEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: LeilSaunaConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up Saunum Leil Sauna climate entity."""
+    coordinator = entry.runtime_data
+    async_add_entities([LeilSaunaClimate(coordinator)])
+
+
+class LeilSaunaClimate(LeilSaunaEntity, ClimateEntity):
+    """Representation of a Saunum Leil Sauna climate entity."""
+
+    _attr_name = None
+    _attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
+    _attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
+    _attr_temperature_unit = UnitOfTemperature.CELSIUS
+    _attr_min_temp = MIN_TEMPERATURE
+    _attr_max_temp = MAX_TEMPERATURE
+
+    def __init__(self, coordinator: LeilSaunaCoordinator) -> None:
+        """Initialize the climate entity."""
+        super().__init__(coordinator, "climate")
```

Comment:
IMO the climate suffix in the unique id isnt needed as the unique id is unique per integration per platform, so a suffix will only be needed when you have multiple climate entiities